.github/workflows/release.yml

```diff
@@ -0,0 +1,24 @@
+name: "Tests: release process"
+
+on: [pull_request, push]
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2-beta
+        with:
+          node-version: "14"
+      - run: npm install
```

Comment:
i'd prefer to use my personal action for this instead: https://github.com/es-shims/Array.prototype.entries/blob/11dd526c66c943cc91b8335e2cd028d9cbe67766/.github/workflows/node-4%2B.yml#L44-L47

Reply:
k, i tried it out, and since it doesn't support `env` yet, let's land this as-is for now and i'll iterate on it later :-)